ServiceBus/ServiceBus/IProductService.cs

```diff
@@ -27,32 +27,39 @@ public interface IProductService
         SharedLibs.DataContracts.Products GetAllProducts();
 
         /// <summary>
-        /// This method adds product into datasource
+        /// This method adds product into datasource. Defaultly product is not main product.
         /// </summary>
         /// <param name="name">Name of a new product</param>
         /// <param name="price">Price of a new product</param>
         /// <param name="guid">ID of a new prodcut</param>
+        /// <param name="pType">ID of a product type</param>
+        /// <param name="headliner">Is this product supposed to be main product?</param>
         /// <returns>Result object</returns>
         [OperationContract (Name = "AddProduct")]
-        SharedLibs.DataContracts.Result AddProduct(string name, double price, Guid guid);
+        SharedLibs.DataContracts.Result AddProduct(string name, double price, Guid guid, int pType, bool headliner);

```

Comment:
1) Please, use whole names for parameters pType -> productType.
2) Why is product type an integer? I would rather see enumeration.